lib/features/family/features/giving_flow/screens/choose_amount_slider_screen.dart

```diff
@@ -51,84 +55,119 @@ class ChooseAmountSliderScreen extends StatelessWidget {
         }
       },
       builder: (context, state) {
-        return Scaffold(
-          appBar: AppBar(
-            toolbarHeight: flow.isQRCode || flow.isRecommendation ? 85 : null,
-            automaticallyImplyLeading: false,
-            leading: const GivtBackButton(),
-            actions: [_getAppBarAction(flow)],
-          ),
-          body: SafeArea(
-            child: Column(
-              children: [
-                Expanded(
-                  child: Column(
-                    children: [
-                      OrganisationWidget(organisation),
-                      const Spacer(),
-                      const BodyMediumText(
-                        'How much would you like to give?',
-                      ),
-                      const SizedBox(height: 32),
-                    ],
-                  ),
-                ),
-                SliderWidget(state.amount, state.maxAmount),
-                const Spacer(),
-              ],
-            ),
+        return FunScaffold(
+          appBar: FunTopAppBar(
+            title: collectgroup.name,
+            leading: const GivtBackButtonFlat(),
+            actions: [
+              showActionIcon(collectgroup),
```

Comment:
Can we please follow flutter naming convention? this "show" stuff really irks me 😂 Especially for declarative stuff, opinion?

Reply:
haha, it was used in different places in this way, so I'll just continue 😂

What would you prefer? 
just actionIcon() ? :)

Reply:
actionIcon definitely!

Reply:
For me show just makes more sense as an "action". Like showbottomSheet, showPopUp etc

Reply:
Thanks! it's changed